scripts/configure_qemu.sh

```diff
@@ -71,7 +71,7 @@ set -x
   --disable-debug-info \
   --disable-glusterfs \
   --cross-prefix=$(xx-info)- \
-  --host-cc=$(xx-clang --print-target-triple)-clang \
+  --host-cc=$(TARGETPLATFORM= TARGETPAIR= xx-clang --print-target-triple)-clang \
   --host=$(xx-clang --print-target-triple) \
   --build=$(TARGETPLATFORM= TARGETPAIR= xx-clang --print-target-triple) \
```

Comment:
Looking at https://github.com/qemu/qemu/blob/7faf9d2f12ace4c1d04cf1a2b39334eef9a45f22/configure#L714-L723

I think we can get rid of `--host` and `--build`

Reply:
Done :)